### DISCUSSION TITLE
joining jmeter-plugins.org

### DISCUSSION THREAD

**Discussion by pmirek** [zalopay-oss/jmeter-grpc-request#29] [General] [answered]
Your plugin is cool. Have you considered adding reference to it in https://jmeter-plugins.org/ ?
This is great place for sharing JMeter plugins. It took me a while to determine the best plugin for GRPC for jmeter, until I found your plugin.
Having it under suite from jmeter-plugins.org and possibility to add it directly from jmeter plugin manager can make your plugin easy to find and use. Can you consider this move now or in the future? Some time ago I had a similar talk with Jose Luis, creator of another cool plugin (https://github.com/JoseLuisSR/mqmeter) and now it's available under jmeter plugins.

Thanks for your great work and easy-to-use plugin!

**Comment by huynhminhtan** [comment, marked as answer]
Hi @pmirek,

The JMeter gRPC Request was published at JMeter Plugins Manager by merge request [jmeter-plugins/pull/458](https://github.com/undera/jmeter-plugins/pull/458). So,
we can find here https://jmeter-plugins.org/?search=jmeter-grpc-request

Thank you
